typescript/tsconfig.node.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "ES2020",
+    "lib": ["ES2020"],
```

Comment:
ES2020 is for node 14 but it will go EOL in a few days so I suggest using ES2021 instead